src/dotnet/Users.UI.Blazor/Components/OwnAvatarEditorModal.razor

```diff
@@ -40,7 +40,7 @@
             </button>
         </FormSection>
 
-        <button type="button" class="delete-avatar-btn">
+        <button type="button" class="delete-avatar-btn" onclick="@OnDeleteAvatarClick">
```

Comment:
Like here, and btw, JS-bound events are also typically "highlighted" with "@onclick" vs "onclick".

Reply:
done